specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementPortalSettingsUpdateDelegation.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2017-03-01",
+    "subscriptionId": "subid",
+    "uid": "5931a75ae4bbd512288c680b",
+    "If-Match": "*",
+    "parameters": {
+      "properties": {
+        "url": "http://contoso.com/delegation",
+        "validationKey": "nVF7aKIvr9mV/RM5lOD0sYoi8ThXTRHQP7o66hvUmjCDkPKR3qxPu/otJcNciz2aQdqPuzJH3ECG4TU2yZjQ7Q==",
```

Comment:
Should this key be public?

Reply:
Yes, as it's not the actual key, but some hashed value of the URL.